pax/config/XENON1T.ini

```diff
@@ -244,6 +244,9 @@ s1_patterns_file =                    'XENON1T_s1_xyz_patterns_Feb2017.json.gz'
 s2_patterns_file =                    'XENON1T_s2_xy_patterns_top_updatedgeo.json.gz'
 s2_mean_area_fraction_top =           0.56             # See XeAnalysisScripts/mapmaking
 
+# Probability of oduble photoelectron emission
```

Comment:
Nah oduble photoelectron emission is a special magic process, don't you know ;-)